src/main/java/com/spotify/docker/client/messages/ServiceCreateResponse.java

```diff
@@ -36,8 +40,21 @@
   @JsonProperty("ID")
   public abstract String id();
 
+  @Nullable
+  @JsonProperty("Warnings")
+  public abstract ImmutableList<String> warnings();
+
   @JsonCreator
-  static ServiceCreateResponse create(@JsonProperty("ID") final String id) {
-    return new AutoValue_ServiceCreateResponse(id);
+  static ServiceCreateResponse create(
+          @JsonProperty("ID") final String id) {
+    return create(id, null);
+  }
+  
+  static ServiceCreateResponse create(
```

Comment:
@johnflavin That's interesting, the test "testCreateServiceWithWarnings" does seem to return the "warnings" content even though the JsonCreator is not on the create with all parameters.